config/routes.rb

```diff
@@ -1,4 +1,11 @@
 Rails.application.routes.draw do
+  get 'errors/file_not_found'
+
+  get 'errors/unprocessable'
+
+  get 'errors/internal_server_error'
+
+  match '/404', to: 'errors#file_not_found', via: :all
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.